packages/@react-stately/selection/src/SelectionManager.ts

```diff
@@ -412,4 +438,17 @@ export class SelectionManager implements MultipleSelectionManager {
 
     return true;
   }
+
+  canSelectItem(key: Key) {
+    if (this.state.selectionMode === 'none' || this.state.disabledKeys.has(key)) {
+      return false;
+    }
+
+    let item = this.collection.getItem(key);
+    if (!item || (item.type === 'cell' && !this.allowsCellSelection)) {
```

Comment:
Looks like this is the thing blocking ListView item selection after the merge from main. The changes from main (aka the stuff from https://github.com/adobe/react-spectrum/pull/2325/files) made it so that we rely on the press handlers on the cell to toggle selection but this will block the handlers from making it to the cell via the changes in useSelectableItem

Reply:
not entirely sure why the ListView row press handlers aren't getting the press event if the ListView cell doesn't have it's press handlers returned